include/sound/soc-dpcm.h

```diff
@@ -156,9 +156,12 @@ enum snd_soc_dpcm_state
 void snd_soc_dpcm_be_set_state(struct snd_soc_pcm_runtime *be, int stream,
 	enum snd_soc_dpcm_state state);
 
+/* update audio routing between PCMs and any DAI links */
```

Comment:
@plbossart I'm documenting this in detail in a Design document. @lgirdwood suggested to prepare a separate PR with only these ALSA patches, related to routing updates. In principle all those patches will aim at this one goal: making it possible to update DPCM routing from virtual SOF and other potential users. It will be a bit difficult, because the "theory" behind all those patches will be the same and it will be silly to copy it in every patch. I'll see how best to make each patch clear on the one hand and avoid copying of the general description on the other.